### PR TITLE
added tests for Violations class

### PR DESCRIPTION
**Pull request by AlessandroMinoccheri**

### CODE REVIEW

File: tests/Unit/Rules/ViolationsTest.php [hunk 1]
```diff
@@ -72,4 +72,36 @@ public function test_to_string(): void
 
         $this->assertEquals($expected, $this->violationStore->toString());
     }
+
+    public function test_get_iterable(): void
```
Comment (fain182) [inline tests/Unit/Rules/ViolationsTest.php:76]: I suppose that the type checker already check this?

Comment (AlessandroMinoccheri) [reply tests/Unit/Rules/ViolationsTest.php:76]: Yes, I was trying to explicit it well. I would like to be more specific, any suggestions @fain182 ?

Comment (AlessandroMinoccheri) [reply tests/Unit/Rules/ViolationsTest.php:76]: @fain182 I changed the assertion with a more proper one, what do you think?

Comment (fain182) [reply tests/Unit/Rules/ViolationsTest.php:76]: much better :+1:

**Review by fain182 — APPROVED** [review]

**Pull request merged**